src/yup.test.ts

```diff
@@ -140,21 +132,9 @@ describe('yupResolver', () => {
       };
       const resolve = await yupResolver(schema)(data);
       expect(await yupResolver(schema)(data)).toMatchSnapshot();
-      expect(resolve.errors['age'].types).toMatchInlineSnapshot(`
-        Object {
-          "typeError": "age must be a \`number\` type, but the final value was: \`NaN\` (cast from the value \`\\"test\\"\`).",
-        }
-      `);
-      expect(resolve.errors['createdOn'].types).toMatchInlineSnapshot(`
-        Object {
-          "typeError": "createdOn must be a \`date\` type, but the final value was: \`Invalid Date\`.",
-        }
-      `);
-      expect(resolve.errors['password'].types).toMatchInlineSnapshot(`
-        Object {
-          "required": "password is a required field",
-        }
-      `);
+      expect(resolve.errors.age.types).toMatchInlineSnapshot(`undefined`);
+      expect(resolve.errors.createdOn.types).toMatchInlineSnapshot(`undefined`);
+      expect(resolve.errors.password.types).toMatchInlineSnapshot(`undefined`);
```

Comment:
Just for 3 of them `expect(resolve.errors.password.types).toBeUndefined()` finally.

Reply:
I will fix this 👍

Reply:
done.